pegasus/sites.v3/code.org/public/student/elementary.haml

```diff
@@ -7,12 +7,14 @@ theme: responsive_full_width
 
 = view :poetry_banner
 
-%section.no-padding-bottom
+%section
```

Comment:
Given the following section only appears in English, is there weird padding in non-EN? I haven't tested, just curious.

Reply:
I added a 3rem top margin to the banner instead. Thanks!